src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+(function (janela) {
+  /**
+   * {virtualclass1} {virtualclass2} are elements you passes with getString function eg:-
+   *   virtualclass.lang.getString('operaBrowserIssue', ['opeara', 27]);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected linebreaks to be 'LF' but found 'CRLF'.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482209)